service-front/package-lock.json

```diff
@@ -10,209 +10,356 @@
       "hasInstallScript": true,
       "license": "MIT",
       "dependencies": {
+        "grunt-contrib-watch": "^1.1.0",
```

Comment:
Is this file actually meant to be commited? I think you might have the same problem as me, that package.json keeps getting regenerated locally.  @townxelliot has made another PR to stop that happening, so he may comment here.

Reply:
It was meant to be but it was a decision I regret, I didn't think the branch would end up hitting stumbling blocks and be delayed. At the time it seemed like we should update the lock file to keep it up to date because it would keep regenerating locally as well like yours, but this PR dragged on for so long that it was a relic from before @townxelliot fixed it. To fix the conflicts, I removed the lock file, re-ran `npm install .` then committed the file it generated instead

Reply:
👍